README.md

```diff
@@ -9,16 +9,16 @@ It is deployed as a web application in a WebLogic Server (WLS) instance, version
 ## Downloading the release
 
 You can find all the releases on the [releases page](https://github.com/oracle/weblogic-monitoring-exporter/releases).  
-To download the web application wls-exporter.war and put your configuration file into the WAR, download the getXXX.sh script and then run:
+To download the web application `wls-exporter.war` and put your configuration file into the WAR, download the `getXXX.sh` script and then run:
 
 ```
 bash getXXX.sh <your-config-file>
 ```
 
 ## Building from source
 
-Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where <version> 
-is the Maven-assigned version number, It is intended for release to Maven Central.
+Use `mvn install` to build the web application. This will create `wls-exporter-<version>`, where _version_
+is the Maven-assigned version number; it is intended for release to Maven Central.
```

Comment:
is the Maven-assigned version number defined in the `pom.xml`.  

I would delete "it is intended for release to Maven Central"

Reply:
Done.